Bio/SearchIO/HmmerIO/hmmer2_text.py

```diff
@@ -321,22 +321,22 @@ class Hmmer2TextIndexer(_BaseHmmerTextIndexer):
     """Indexer for hmmer2-text format."""
 
     _parser = Hmmer2TextParser
-    qresult_start = _as_bytes("Query")
+    qresult_start = b"Query"
     # qresults_ends for hmmpfam and hmmsearch
     # need to anticipate both since hmmsearch have different query end mark
-    qresult_end = _as_bytes("//")
+    qresult_end = b"//"
 
     def __iter__(self):
         """Iterate over Hmmer2TextIndexer; yields query results' key, offsets, 0."""
         handle = self._handle
         handle.seek(0)
         start_offset = handle.tell()
-        regex_id = re.compile(_as_bytes(r"Query\s*(?:sequence|HMM)?:\s*(.*)"))
+        regex_id = re.compile(br"Query\s*(?:sequence|HMM)?:\s*(.*)")
```

Comment:
`b"Query\s*(?:sequence|HMM)?:\s*(.*)"`

Reply:
No we need the raw prefix as there are slashes in this string.

Reply:
python thinks it's OK:
```
>>> br"Query\s*(?:sequence|HMM)?:\s*(.*)" == b"Query\s*(?:sequence|HMM)?:\s*(.*)"
True
>>> 
```

Reply:
Yes, but that is changing in later Python 3 - see other thread of review.